tests/unit/utils/test_fs.py

```diff
@@ -149,3 +150,10 @@ def test_relpath_windows_different_drives():
     rel_info = relpath(info1, info2)
     assert isinstance(rel_info, str)
     assert rel_info == path1
+
+
+def test_get_inode(repo_dir, dvc_repo):
+    # Mocking a file to get the inode.
```

Comment:
You are not mocking anything in this test
```suggestion
```

Reply:
@efiop I am creating a file `FOO` so that I can *mock* a path. Isn't this called mocking?

Reply:
Mocking is when you simulate something, and here you are testing on a real file, so I wouldn't call that mocking.

Reply:
@efiop Oh okay. But how does it delete the file later?

Reply:
@efiop Can you answer this, please? :)

Reply:
@algomaster99 Sorry, missed this. Yes, it deletes it. See dvc_repo fixture in tests/conftest.py.